book/src/SUMMARY.md

```diff
@@ -17,6 +17,7 @@
 - [Review file markups](./review.md)
     - [Prr directive example](./examples/prr_directive.md)
     - [Review comment example](./examples/review_comment.md)
+    - [Description comment example](./examples/description_comment.md)
```

Comment:
I assume this file will be added in a later PR?

Reply:
good catch, forgot to git-add it :p